examples/testing-network-shim-app/.eslintignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
I ran into some annoying problems with ESLint rules in this App so I disabled them.

Reply:
See https://github.com/dhis2/cli-utils-cypress/pull/218/commits/5bc2da95146dbea10c1cdbf2751af3ded8ab4743 and b0abf9b. I re-enabled the linting and fixed the annoying lint errors. Hope this is OK with you.

Reply:
Thanks !